src/diffusers/models/attention_processor.py

```diff
@@ -4283,6 +4283,79 @@ def __init__(self):
         super().__init__()
 
 
+class DCAEAttnProcessor:
+    r"""
+    This class is modified from https://github.com/mit-han-lab/efficientvit
+    Processor for EfficientViT in DC-AE.
+    """
+
+    def __call__(
+        self,
+        attn: Attention,
+        hidden_states: torch.Tensor,
+        encoder_hidden_states: Optional[torch.Tensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
+        temb: Optional[torch.Tensor] = None,
+        *args,
+        **kwargs,
+    ) -> torch.Tensor:
+        if len(args) > 0 or kwargs.get("scale", None) is not None:
+            deprecation_message = "The `scale` argument is deprecated and will be ignored. Please remove it, as passing it will raise an error in the future. `scale` should directly be passed while calling the underlying pipeline component i.e., via `cross_attention_kwargs`."
+            deprecate("scale", "1.0.0", deprecation_message)
```

Comment:
This is a new class so, we can just remove this completely 
```suggestion
```

Reply:
Sure. Thanks for pointing out!